.rubocop-https---raw-githubusercontent-com-standout-Coding-master-ruby-rubocop-yml

```diff
@@ -0,0 +1,40 @@
+AllCops:
```

Comment:
But offline development will still be available because if you put this file in `.gitignore` you will still get an lokal version when running `rubocop` first time. And it is already tracked in it's own git repo. If anyone needs to lock a version they can do so by inherit from an commit hash like this

```yaml
inherit_from:
  - https://raw.githubusercontent.com/standout/Coding/99390675b3f51a7efb4f297b97c71cfff39a4586/ruby/rubocop.yml
```